### ISSUE TITLE
[🐛 Bug]: Mocha root hook plugins not working - Problem with `require` arg

### ISSUE THREAD

**Issue by m4hdyar** [webdriverio/webdriverio#9490]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

latest

### Node.js Version

16.13.2

### Mode

WDIO Testrunner

### Which capabilities are you using?

```typescript
Using electron and windows 10
```

### Reproduce
1. Init a project
2. Add Mocha framework
3. Create a [Mocha root hook plugin](https://mochajs.org/#root-hook-plugins) :
```
console.log('Hi)// **You'll see this in console**
export const mochaHooks = {
  beforeEach : async function () {
    console.log("Before each in plugin root hooks"); // But not this line
  }
};
```

4. Add it with `require` option to mochaOpts in `wdio.conf.ts`:

```
mochaOpts: {
  ui: 'bdd',
  timeout: 30000,
  require: './test/aroothook.ts'
},
```
### Problem 
It doesn't work. No error. 

### Another problem that has a workaround:
**require** option has also another bug. If you use it in windows you'll face this error:
`node_modules/@wdio/mocha-framework/build/common.js:120
throw new Error(Module ${name} can't get loaded. Are you sure you have installed it?\n`
`Error: Module test/aroothooks.ts can't get loaded. Are you sure you have installed it?
`
The reason is in Windows you need `File url` when you want to use import() 
`node_modules/@wdio/mocha-framework/build/common.js` line 117.
        return await import(name);
Where the name is the absolute path starting with c: (in my case)

There **is** a workaround for this problem: use relative path.
Example : require: '../../../../test/aroothook.ts'

### How to reproduce the bug.

Steps are above.

### Relevant log output

```typescript
2022-12-29T14:29:38.267Z DEBUG electron-service: launcher received options: {
  ...
  appN2022-12-29T14:29:38.620Z DEBUG @wdio/cli:utils: Finished to run "onPrepare" hook in 350ms
2022-12-29T14:29:38.622Z INFO @wdio/cli:launcher: Run onWorkerStart hook
2022-12-29T14:29:38.622Z DEBUG @wdio/cli:utils: Finished to run "onWorkerStart" hook in 0ms
2022-12-29T14:29:38.623Z INFO @wdio/local-runner: Start worker 0-0 with arg: .\\wdio.conf.ts
2022-12-29T14:29:38.630Z DEBUG @wdio/local-runner: Send command run to worker with cid "0-0"
2022-12-29T14:29:47.914Z DEBUG @wdio/local-runner: Runner 0-0 finished with exit code 0
2022-12-29T14:29:47.915Z INFO @wdio/cli:launcher: Run onWorkerEnd hook
2022-12-29T14:29:47.915Z DEBUG @wdio/cli:utils: Finished to run "onWorkerEnd" hook in 0ms
2022-12-29T14:29:47.915Z INFO @wdio/cli:launcher: Run onWorkerStart hook
2022-12-29T14:29:47.915Z DEBUG @wdio/cli:utils: Finished to run "onWorkerStart" hook in 0ms
2022-12-29T14:29:47.917Z INFO @wdio/local-runner: Start worker 0-1 with arg: .\\wdio.conf.ts
2022-12-29T14:29:47.925Z DEBUG @wdio/local-runner: Send command run to worker with cid "0-1"
2022-12-29T14:29:52.954Z DEBUG @wdio/local-runner: Runner 0-1 finished with exit code 0
2022-12-29T14:29:52.954Z INFO @wdio/cli:launcher: Run onWorkerEnd hook
2022-12-29T14:29:52.955Z DEBUG @wdio/cli:utils: Finished to run "onWorkerEnd" hook in 1ms
2022-12-29T14:29:52.955Z INFO @wdio/cli:launcher: Run onComplete hook
2022-12-29T14:29:52.955Z DEBUG @wdio/cli:utils: Finished to run "onComplete" hook in 0ms
2022-12-29T14:29:52.983Z INFO @wdio/local-runner: Shutting down spawned worker
2022-12-29T14:29:53.247Z INFO @wdio/local-runner: Waiting for 0 to shut down gracefully
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by christian-bromann** [comment]
Hey,
thanks for filing the issue. This seems indeed like bug. We would appreciate any help on this so please help us out! Have a look at our [contributing guidelines](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) and reach out if you have any questions. Cheers!

**Comment by christian-bromann** [comment]
Implemented in #9663
